src/containers/TaxonomyVersions/TaxonomyVersionsPage.tsx

```diff
@@ -60,6 +87,16 @@ const TaxonomyVersionsPage = () => {
             {t('taxonomyVersions.newVersionButton')}
           </ButtonV2>
         </Row>
+
+        {publishRequests?.length ? (
+          <>
+            <h2>{t('publishRequests.deleteAll')}</h2>
+            <DangerZone>
```

Comment:
:skull:

Reply:
Denne sida er åpen for alle, så kanskje vi kan skjule heile dette valget for alle som ikkje er admin.

Reply:
Ja, jeg og. Kan alltids flytte den.

Reply:
Lars tenker den kan ligge på denne sida fordi den skal brukes så sjelden, men legg den nederst.